src/lib/FormGroupLayout.tsx

```diff
@@ -36,7 +38,15 @@ const FormGroupLayout = <T extends FieldValues>(props: FormGroupLayoutProps<T>)
   ) : (
     <FormGroup switch={switchLayout ? true : undefined} check={checkboxLayout ? true : undefined}>
       <FormGroupLayoutLabel<T> label={label} fieldName={name} fieldId={id} tooltip={labelToolTip} layout={layout} />
-      {children}
+      <InputGroup
```

Comment:
@neotob

Not sure if it's a problem to always render the `InputGroup`?

Reply:
i dont see this as a problem... as its only a visual helper... question for me is more a little what about the inputOnly?

Reply:
@neotob 

Ah you think the addons should be rendered with input only?

Reply:
@neotrow 

i was debating about it, but i decided now that it shouldn't :)